content/docs/hooks-intro.md

```diff
@@ -61,7 +61,7 @@ React Conference 2018 ൽ വച്ച് സോഫി അൽപ്പർട്
 
 ### Components തമ്മിൽ ലോജിക് പുനരുപയോഗിക്കാൻ ഉള്ള ബുദ്ധിമുട്ട് {#its-hard-to-reuse-stateful-logic-between-components}
 
-ഇത് വരെ React ൽ Components ലെക് പുനരുപയോഗിക്കാവുന്ന ലോജിക് ബന്ധിപ്പിക്കാൻ ഉള്ള മാർഗം ഉണ്ടായിരുന്നില്ല. [റെൻഡർ props](/docs/render-props.html) അല്ലെങ്കിൽ [Higher Order Components](/docs/higher-order-components.html) പോലുള്ള മാതൃകകൾ നിങ്ങൾ നേരത്തെ ശ്രദ്ധിച്ചു കാണും. പക്ഷെ ഇവ ഉപയോഗിക്കുമ്പോൾ Components പുനർരൂപീകരിക്കേണ്ടി വരും എന്നത് ഒരു മുഖ്യമായ ഒരു പോരായ്മ ആയിരുന്നു. ഇത് കൂടാതെ, ഇവ "Wrapper Hell" എന്ന് വിശേഷിപ്പിക്കാവുന്ന ഒരു പ്രതിഭാസം കൂടി ഒപ്പം കൊണ്ട് വന്നു. റെൻഡർ Props,Higher Order Components എന്നിവ ഉപയോഗിക്കുമ്പോൾ React Dev Tools ൽ നിങ്ങൾക്കു കാണാൻ ആകുന്നത് DOM എലെമെന്റ്സിന് ഒപ്പം കുറെ  പുറംചട്ടകൾ കൂടി വരുന്നത്. ഇവ debugging അനുഭവം ബുദ്ധിമുട്ടേറിയതു ആക്കി. ഈ പുറം ചട്ടകളെ [ഫിൽറ്റർ ചെയ്തു മാറ്റാൻ വഴികൾ](https://github.com/facebook/react-devtools/pull/503) ഉണ്ട് എങ്കിലും,  പ്രശനം വ്യക്തം ആയിരുന്നു: ലോജിക് ആയോജിപ്പിക്കാൻ React ൽ മെച്ചപ്പെട്ട പ്രിമിറ്റീവ്സ് ആപേക്ഷിതം ആണ്. 
+ഇത് വരെ React ൽ Components ലെക് പുനരുപയോഗിക്കാവുന്ന ലോജിക് ബന്ധിപ്പിക്കാൻ ഉള്ള മാർഗം ഉണ്ടായിരുന്നില്ല. [റെൻഡർ props](/docs/render-props.html) അല്ലെങ്കിൽ [Higher Order Components](/docs/higher-order-components.html) പോലുള്ള മാതൃകകൾ നിങ്ങൾ നേരത്തെ ശ്രദ്ധിച്ചു കാണും. പക്ഷെ ഇവ ഉപയോഗിക്കുമ്പോൾ Components പുനർരൂപീകരിക്കേണ്ടി വരും എന്നത് മുഖ്യമായ ഒരു പോരായ്മ ആയിരുന്നു. ഇത് കൂടാതെ, ഇവ "Wrapper Hell" എന്ന് വിശേഷിപ്പിക്കാവുന്ന ഒരു പ്രതിഭാസം കൂടി ഒപ്പം കൊണ്ട് വന്നു. റെൻഡർ Props,Higher Order Components എന്നിവ ഉപയോഗിക്കുമ്പോൾ React Dev Tools ൽ നിങ്ങൾക്കു കാണാൻ ആകുന്നത് DOM എലെമെന്റ്സിന് ഒപ്പം കുറെ  പുറംചട്ടകൾ കൂടി വരുന്നത്. ഇവ debugging അനുഭവം ബുദ്ധിമുട്ടേറിയതു ആക്കി. ഈ പുറം ചട്ടകളെ [ഫിൽറ്റർ ചെയ്തു മാറ്റാൻ വഴികൾ](https://github.com/facebook/react-devtools/pull/503) ഉണ്ട് എങ്കിലും,  പ്രശനം വ്യക്തം ആയിരുന്നു: ലോജിക് ആയോജിപ്പിക്കാൻ React ൽ മെച്ചപ്പെട്ട പ്രിമിറ്റീവ്സ് ആപേക്ഷിതം ആണ്. 
 
 Hooks ഉപയോഗിച്ചു കൊണ്ട് നിങ്ങൾക്കു stateful ലോജിക് component ൽ നിന്ന് വേർതിരിച്ചു എഴുതാം. ഇതിലൂടെ നിങ്ങൾക്കു ഇത് ടെസ്റ്റ് ചെയാനും പുനരുപയോഗിക്കാനും സാധിക്കും. **Hooks ഉപയോഗിച്ച് Components ന്റെ അധികാരക്രമം മാറ്റാതെ തന്നെ നിങ്ങൾക്കു ലോജിക് പുനരുപയോഗിക്കാൻ ആവും വിധം എഴുതാൻ സാധിക്കും.** ഇത് കൊണ്ട് തന്നെ നിങ്ങൾ എഴുതുന്ന Hooks നിങ്ങളുടെ പ്രോജെക്ടിലെ മറ്റു components ഉം ആയിട്ടോ, മറ്റുള്ളവരും ആയിട്ടോ പങ്ക് വക്കാൻ എളുപ്പം ആണ്.
```

Comment:
For better readability, I'm suggesting a couple of changes. @BoyWithSilverWings 

പ്രശനം വ്യക്തം ആയിരുന്നു - പ്രശ്നം. 
പങ്ക് വക്കാൻ - പങ്കുവെയ്ക്കാന്‍
മറ്റുള്ളവരും ആയിട്ടോ - മറ്റുള്ളവരുമായോ
ഇവ debugging - ഇത് debugging
ഇത് ടെസ്റ്റ് ചെയാനും - ഇത് ടെസ്റ്റ് ചെയ്യുവാനും
ഉണ്ട് എങ്കിലും - ഉണ്ടെങ്കിലും

Reply:
Changes made